packages/auth/migrations/20220504161932_create_grants_table.js

```diff
@@ -5,18 +5,18 @@ exports.up = function (knex) {
     table.string('state').notNullable()
     table.specificType('startMethod', 'text[]').notNullable()
 
-    table.string('continueToken').unique()
-    table.string('continueId').unique()
+    table.string('continueToken').notNullable().unique()
+    table.string('continueId').notNullable().unique()
```

Comment:
It shouldn't be necessary to generate continuation information if interaction isn't necessary for a grant. The `POST /` should just return the access token without any continuation information.

Reply:
I think we discussed that while you were away. We always want to be able to delete/revoke the grant and for that you need the continue token and id. Hence, it must always be present. We also already changed the spec to make it always required.